### PR TITLE
Changed Links and added buttons

### PR DESCRIPTION
**Pull request by hariarun18**

### CODE REVIEW

File: landing/config/_default/params.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ image = "images/symphony_icon_largetext.png" # this image will be used as fallba
 # contact form action
 contact_form_action = "#" # contact form works with https://formspree.io
 
-# Type of pages to be rendered with a blog view
+# Type of pages to be rendered with a blog vie
```
Comment (Haishi2016) [inline landing/config/_default/params.toml:12]: typo - vie(w)

**Review by Haishi2016 — CHANGES_REQUESTED** [review]
Please also accept license agreement before this PR can be accepted.

File: landing/config/_default/params.toml (hunk 1 above)
Comment (hariarun18) [reply landing/config/_default/params.toml]: Spelling issue fixed in 10/20 commit

**Comment by hariarun18** [comment]
Have completed the ECA agreements - not reflecting here, if this prevents this from moving forward I can follow up with Eclipse team. OCX 2024 Blocker.

**Review by Haishi2016 — APPROVED** [review]

**Pull request merged**